### DISCUSSION TITLE
Event log generating rate (clarification)

### DISCUSSION THREAD

**Discussion by gutleib** [seaweedfs/seaweedfs#6295] [Q&A]
Hi!
As I understood after reading [this discussion](https://github.com/seaweedfs/seaweedfs/discussions/4727) and [filer replication document](https://github.com/seaweedfs/seaweedfs/wiki/Filer-Store-Replication), each event on FUSE mount generates log entry, that is replicated to other filers, so they can server new metadata of the filesystem. AFAIU, this creates rather large amount of data, as in gigabytese per day for filesystem containing database files. Wow!

Is to also true for S3 filers? Does any access to object generate event, or just an "write or other "modify" occurrence?
Is it true for shared storages? Do they have event log at all?

**Comment by chrislusf** [comment]
Only create or update operations are in the logs.

**Comment by gutleib** [comment]
Thank you
So, S3 generate same amount of logs. 
And what about shared filer-DB?
